api-client/src/models/TunewsModel.ts

```diff
@@ -17,8 +18,11 @@ class TunewsModel {
     eNewsNo: string
   }) {
     const { id, date, title, tags, content, eNewsNo } = params
+
+    const parsedTitle = parseHTML(title, {})
```

Comment:
You could make the second param optional so you don't have to pass an empty object all the time.

Reply:
But shouldn't we use `decodeEntities` here anyway?